__tests__/no-path-versioning.test.ts

```diff
@@ -27,23 +27,15 @@ testRule("no-path-versioning", [
         "/v12/foobar": {},
         "/v1.2/foobar": {},
         "/v1.2.3/foobar": {},
-        "/1.0/foobar": {},
```

Comment:
this does look like a version, I'm not sure we shouldn't be treating them as if they weren't.

I'll try tweaking the regexp a bit to see if we can fix the use case from the bug report while preserving the existing behavior

Reply:
I mean it kinda does but whats to say this is a version and not a number like we have in our mock servers. 🤔